sql/core/src/test/scala/org/apache/spark/sql/SQLQuerySuite.scala

```diff
@@ -3336,6 +3336,35 @@ class SQLQuerySuite extends QueryTest with SharedSparkSession {
       checkAnswer(df5, Array.empty[Row])
     }
   }
+
+  test("SPARK-29505: desc columnname - case insensitive search") {
```

Comment:
Could you try to move this test to `describe-table-column.sql`? You can use SQL there. Please refer `SQLQueryTestSuite` for how to use.

Reply:
+1 for the move.

Reply:
@dongjoon-hyun and @maropu i have addressed the review comments could you kindly verify?